build_config/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.2.3-dev
+
+- Expose key normalization methods publically, these include:
+  - normalizeBuilderKeyDefinition
```

Comment:
[optional] I think only they `*Usage` methods should be necessary outside this package.

Reply:
While I generally agree it felt weird to only export some of them, and I don't think it is likely to cause any more churn to expose them all as they all essentially have the same api.

Reply:
changed my mind lol gonna actually remove these